applications/nrf5340_audio/doc/feature_support.rst

```diff
@@ -21,7 +21,19 @@ For an explanation of the maturity levels, see :ref:`Software maturity levels <s
 nRF5340 Audio QDIDs
 *******************
 
-The following QDIDs are related to the nRF5340 Audio application:
+The following DNs/QDIDs are related to the nRF5340 Audio application:
+
+nRF5340 DK Bluetooth DNs/QDIDs
+   A full Audio product QDID requires several DNs and QDIDs for the nRF5340 DK, as listed in the `nRF5340 DK Bluetooth DNs and QDIDs Compatibility Matrix`_.
+   The exact DN/QDID numbers depend on the project configuration and the features used in the application.
+   They might include DNs/QDIDs for the SoftDevice Controller subsystem, Host subsystem, or LE Audio Profiles and Services.
+   These in turn can depend on ISO transport and the SoC version.
```

Comment:
1) How does this depend upon the SoC version? After talking to Miles, I believe this is not the case.
("That would create an unholy mess. He should be headslapped for even saying those words.") :-)

2) (There does not seem to be such a thing as SoC versions anyway - it is "SoC revisions and variants" - https://docs.nordicsemi.com/bundle/comp_matrix_nrf5340/page/COMP/nrf5340/nrf5340_ic_revision_overview.html )